### PR TITLE
fixes for iv page

### PR DESCRIPTION
**Pull request by Rahien**
## Description

Make the installatievergadering page update when the burgemeester is changed.
Show warnings about burgemeester in the college orgaan the same way as it is done for bcsd

## How to test

Change the mayor, see the college and vast bureau roles change
Add a true mayor, see an error
Add an extra aangewezen burgemeester, see an error

### CODE REVIEW

File: app/components/mandaat/burgemeester-selector.js [hunk 1]
```diff
@@ -17,14 +17,14 @@ import {
 import { toUserReadableListing } from 'frontend-lmb/utils/to-user-readable-listing';
```
Comment (JonasVanHoof) [inline app/components/mandaat/burgemeester-selector.js:17]: Needs to be removed for lint job to succeed*

File: app/components/mandaat/burgemeester-selector.js [hunk 2]
```diff
@@ -17,14 +17,14 @@ import {
...
 import { restartableTask } from 'ember-concurrency';
 import { getFormFrom } from 'frontend-lmb/utils/get-form';
+import { CBS_BESTUURSORGAAN_URI } from 'frontend-lmb/utils/well-known-uris';
```
Comment (JonasVanHoof) [inline app/components/mandaat/burgemeester-selector.js:20]: Needs to be removed for lint job to succeed*

File: app/components/verkiezingen/burgemeester-alert.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  @tracked lastRecomputeTime = null;
+
+  constructor() {
+    super(...arguments);
+    this.bcsd.forceRecomputeBCSD();
+  }
+
+  get collegeOrgaanInTijd() {
+    return this.args.collegeBestuursorgaanInTijd;
+  }
```
Comment (JonasVanHoof) [inline app/components/verkiezingen/burgemeester-alert.js:25]: Not really necessary imo

**Review by JonasVanHoof — APPROVED** [review]
Looks good, do we also need to add a warning that they have added two aangewezen burgemeesters?
image

File: app/components/verkiezingen/burgemeester-alert.js (hunk 3 above)
Comment (Rahien) [reply app/components/verkiezingen/burgemeester-alert.js:24]: i sortof like this pattern for documentation purposes, this way you see better that there is an arg

**Comment by Rahien** [comment]
> Looks good, do we also need to add a warning that they have added two aangewezen burgemeesters? 

it's there but the burgemeester one takes prio

**Pull request merged**